packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -11,6 +22,8 @@ import './my-profile.scss';
 
 const MyProfile = observer(() => {
     const { general_store, my_profile_store } = useStores();
+    const [on_screen_keyboard, toggleOnScreenKeyboard] = React.useState(false);
```

Comment:
`on_screen_keyboard` is a boolean so it should be named as such, e.g. `has_on_screen_keyboard`, also `toggleOnScreenKeyboard` suggests the function toggles the value of `on_screen_keyboard`, which it doesn't. Try to rename it to something like `setHasOnScreenKeyboard`